code/weapon/beam.h

```diff
@@ -45,9 +43,10 @@ struct vec3d;
 // uses to define beam behavior ahead of time - needed for multiplayer
 typedef struct beam_info {
 	vec3d			dir_a, dir_b;						// direction vectors for beams	
-	float				delta_ang;							// angle between dir_a and dir_b
-	ubyte				shot_count;							// # of shots	
-	float				shot_aim[MAX_BEAM_SHOTS];		// accuracy. this is a constant multiple of radius. anything < 1.0 will guarantee a hit	
+	float			delta_ang;							// angle between dir_a and dir_b
+	ubyte			shot_count;							// # of shots	
+	float			shot_aim[MAX_BEAM_SHOTS];		// accuracy. this is a constant multiple of radius. anything < 1.0 will guarantee a hit
+	vec3d           rot_axis;
```

Comment:
This breaks multi. The `beam_info` struct is sent in it's entirety so any change to it will break compatibility. That is fixed in the multi_packet_fixes PR though. If you can do this in another way, even if it's temporary, then it can be added back safely as part of multi_packet_fixes.

Reply:
There's very little that can be done to avoid sending rot_axis, for the same reasons that dir_a and b need to be sent. The best I can afford is that several other things here in b_info don't actually need to be need to be sent, `delta_ang` at the very least.

In the end, I don't really mind waiting, as I suspected this would require some sort of multi adjustment, although I didn't quite know the nature of it.

Reply:
Well if you don't mind waiting a bit then that solves the problem. I can go ahead and add `rot_axis` as part of the packet fixes and you can just make use it. Then everything should "just work".